Model/Sales/MagentoOrderCollection.php

```diff
@@ -410,85 +498,21 @@ private function createShipment(Order $order)
     }
 
     /**
-     * Update all the tracks that made created via the API
+     * @param object $orderItem
      *
-     * @return $this
-     * @throws \Magento\Framework\Exception\LocalizedException
+     * @return string
      */
-    public function updateMagentoTrack()
+    private function getMultiStockInventory($orderItem): string
```

Comment:
Is er geen type voor `$orderItem`?

Reply:
$orderItem is een object en dat is pas mogelijk vanaf php7.2